turbinia/workers/__init__.py

```diff
@@ -75,6 +77,8 @@ def __init__(
       TurbiniaException: If the Output Manager is not setup.
     """
 
+    if mock:
```

Comment:
This feels a little strange to me.  I wonder if we should move the `output_manager.get_local_output_dirs()` and exception raising to somewhere else like a `setup()` method?  That would probably make __init__ a little cleaner anyway.  I took a brief look, and it seems like there are only a couple places where new `TurbiniaTaskResult()`s are created, so it should be fairly easy to add a setup() that gets called after those instantiations.  WDYT?

Reply:
Yeah, a `setup()` method makes sense.

Reply:
I moved `task` to be an argument to `setup()` instead of `init` and any attributes that require it are now instantiated down there.